### PR TITLE
eslint, husky, prettier, lint-staged deps added

### PR DESCRIPTION
**Pull request by bdvorianov**
Issue #3

### CODE REVIEW

**Comment by bdvorianov** [comment]
+ airbnb config is there

**Comment by bdvorianov** [comment]
+ Conflicts with master branch fixed
Check pre-commiting hook and close. @denysdovhan

File: package.json [hunk 1]
```diff
@@ -1,18 +1,24 @@
 {
-  "name": "chernivtsi-transport",
+  "name": "gps",
```
Comment (denysdovhan) [inline package.json]: Don't change that

File: package.json [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
   "version": "0.1.0",
   "private": true,
-  "workspaces": [
```
Comment (denysdovhan) [inline package.json]: `workspaces` was added lately. Please, keep them. They're required to work with Lerna.

File: package.json [hunk 3]
```diff
@@ -1,18 +1,24 @@
...
-    "api",
-    "app"
-  ],
+  "dependencies": {
+    "express": "^4.16.4",
```
Comment (denysdovhan) [inline package.json]: Why do you touch these files? Those deps were moved to `api`.

File: package.json [hunk 4]
```diff
@@ -1,18 +1,24 @@
...
+    "forever": "^0.15.3",
+    "husky": "^1.3.1",
```
Comment (denysdovhan) [inline package.json]: `husky`, `prettier`, `lint-staged` — these are dev deps.

File: package.json [hunk 5]
```diff
@@ -1,18 +1,24 @@
...
+    "isomorphic-fetch": "^2.2.1",
+    "lint-staged": "^8.1.0",
+    "prettier": "^1.15.3"
+  },
   "scripts": {
-    "start": "lerna run start --stream",
```
Comment (denysdovhan) [inline package.json]: Don't remove these lines. I've added monorepo support lately.

**Comment by bdvorianov** [comment]
Mistakes made by incorrect local merging order, fixing...

**Comment by bdvorianov** [comment]
Mistakes made because of incorrect local merging order, fixing...

**Comment by bdvorianov** [comment]
Check it, @denysdovhan

**Comment by denysdovhan** [comment]
Thanks for contributing!

**Pull request merged**